src/yang/ietf-bgp-types.yang

```diff
@@ -493,7 +493,7 @@ module ietf-bgp-types {
       }
       enum confederation {
         description
-          "Confederation as peer";
+          "Confederation AS peer";
```

Comment:
Missing a period.

Reply:
I have added to the diff, but this leaves it inconsistent vs. other items in the list.

A different issue is opened to address the lack of punctuation throughout the file set.